website/docs/r/kinesis_metrics_source.html.markdown

```diff
@@ -128,4 +128,282 @@ HTTP sources can be imported using the collector name and source name (`collecto
 terraform import sumologic_kinesis_metrics_source.test my-test-collector/my-test-source
 ```
 
+## Full Example (Including terraform for AWS asset creation)
+```hcl
+terraform {
+  required_providers {
+    sumologic = {
+      source = "sumologic/sumologic"
+    }
+    aws = {
+      source = "hashicorp/aws"
+    }
+  }
+}
+
+provider "sumologic" {}
+provider "aws" {}
+
+locals {
+  account_id = ""
+  aws_access_key = ""
+  aws_secret_key = ""
+
+  description = "update your terraform description here"
+  identifier = "SumologicMetricStream"
+
+  region = "us-west-2"
+
+  tagfilters = [
+    { type = "TagFilters", namespace = "AWS/ApplicationELB", tags = ["Deployment=prod"] },
+  ]
+
+}
+
+resource "sumologic_collector" "collector_for_kinesis_metrics" {
+  name = "AWS Metrics via Kinesis"
+}
+
+resource "sumologic_kinesis_metrics_source" "kinesis_source" {
+  name          = "CloudWatch Metrics via Kinesis"
+  description   = "Description for Sumologic source"
+  category      = "aws/cloudwatch"
+  content_type  = "KinesisMetric"
+  collector_id  = sumologic_collector.collector_for_kinesis_metrics.id
+
+  authentication {
+    type = "S3BucketAuthentication"
+    access_key = local.aws_access_key
+    secret_key = local.aws_secret_key
+  }
+
+  path {
+    type = "KinesisMetricPath"
+
+    dynamic "tag_filters" {
+      for_each = local.tagfilters
+      content {
+        type      = tag_filters.value.type
+        namespace = tag_filters.value.namespace
+        tags      = tag_filters.value.tags
+      }
+    }
+  }
+}
+
```

Comment:
should we remove the sumologic resources as they are already mentioned above in the Example usage section?

Reply:
I am thinking if someone just does copy and paste of the full section it should work.
I was working with customer, and I feel copy paste should work for one example end-to-end.